test/powershell/Modules/Microsoft.PowerShell.Utility/Select-String.Tests.ps1

```diff
@@ -1,6 +1,6 @@
 # Copyright (c) Microsoft Corporation. All rights reserved.
 # Licensed under the MIT License.
-Describe "Select-String" -Tags "CI" {
+Describe "Select-String" -Tags "CI","SLS" {
```

Comment:
Why is there a `SLS` tag?  We're using specific tags that our CI understands: https://github.com/PowerShell/PowerShell/blob/master/docs/testing-guidelines/WritingPesterTests.md#describecontextit

Reply:
I'll remove that - was there to shorten my test cycles. Has slipped in by accident.